### DISCUSSION TITLE
Using Multiple EEPROM in a file loading same data- reg

### DISCUSSION THREAD

**Discussion by codeporul** [hneemann/Digital#1313] [Q&A] [answered]
Good day,
When i am using two EEPROM in file , which has different hex values. But while simulating , last loaded hex is displaying as data/output for both EEPROM instead of only one. Looks like simple issue .But i am not able to find out that.help needed.Thank you.

**Comment by hneemann** [comment]
Are you using the two EEPROM built-in components or the 28cXXX library parts?

**Reply by codeporul** [reply]
Yes.Components -> Library -> Dil chips -> eeproms ->28c16

**Reply by hneemann** [reply, marked as answer]
There are two solutions:

1. Copy the Library part to your dig folder twice and load your data in both of them or
2. Give the components a label and set the content via Edit->Circuit specific settings->Advanced->Content of ROM's->Edit

I would prefer the second method.

**Reply by codeporul** [reply]
Update:
Using  2nd option Learned to create component. And now i am using the 2nd method. tested 2nd option in little bit big circuits,small circuits & it is working fine for me. So far , tested 1st method in small circuits & it is working fine.
///Thank you very much. 1st method worked for me.
